data/korath/korath missions.txt

```diff
@@ -257,3 +257,48 @@ mission "Small Scale Delivery"
 	on complete
 		payment 1000
 		dialog `You find a Korath who seems to have been waiting for you in the spaceport. They greet you by showing their palms, then outstretch their hands as if waiting to receive the package. You hand off the small delivery and they hand you a small payment of <payment>.`
+
+mission "Korath Boarding Response"
+	invisible
+	boarding
+	repeat
+	minor
+	deadline 1
+	to offer
+		not "Korath Boarding Response: active"
+		or
+			and
+				random < 5
+				"reputation: Korath" < -10
+			and
+				random < 10
+				"reputation: Korath" < -100
+			and
+				random < 25
+				"reputation: Korath" < -1000
+			and
+				random < 50
+				"reputation: Korath" < -10000
+
+			and
+				random < 10
+				"reputation: Korath (Civilian)" < 0
+			and
+				random < 25
+				"reputation: Korath (Civilian)" < -100
+			and
+				random < 50
+				"reputation: Korath (Civilian)" < -1000
+	source
+		government Korath "Korath (Civilian)"
+		system Alcyone Almach "Alpha Hydri" Atik Durax Misam Polaris Sheratan
+	on offer
+		conversation
+			"An angry hiss comes over your radio, and you see a strange pulse along the hull of the Korath vessel. Your ship detects a powerful hyperwave emission followed by several capital ships jumping in. You had best finish your business with this ship before yours is reduced to a ball of superheated plasma."
+				accept
+	# The destination is mandatory, but its choice won't affect anything due to "deadline 1"
+	destination "Kessel Sepret"
+	npc kill
+		government "Korath"
+		personality harvests plunders unconstrained coward
+		fleet "Korath Defense" 4
```

Comment:
I'd include a to complete never just for the clarity of it, but not needed
Btw we could make the size of this fleet depend on the strength of the player in terms of "armament deterrance"? Surely sending this much against a solo player ship is overkill
And they could send more if you have more
Just a thought

Reply:
Also didnt you forget entering? Thats how it usually happens with these boarding spawning missions

Reply:
This isn't being sent against the player's ship. They rampage across the system killing everything. The player is a target if the player gets in the way.

Reply:
there is just not as fluidly, using to spawn
Could not be required though idk

Reply:
The job of the fleet is to completely wipe the system clean. I am sending the minimum amount of ships the Korath need to do that with 0 casualties at 100% reliability, assuming the player's strength is negligible. If I reduce the fleet size, the Korath will lose ships, and the rescue mission would be pointless.

Reply:
You could increase the amount of ships though if you want to send a message to the player
but for now should be fine as is (even less would wipe the system tbh)

Reply:
I want the player to be able to escape. Also, there were lore objections to the Korath gathering a massive fleet just to send a message to Captain Last. From a logistics standpoint, it makes more sense for them to wipe the system clean, so it is safe to raid again. Certainly, they should send more ships if the system is busier, but I need to add faction-level AI to do that.